.travis.yml

```diff
@@ -13,16 +13,15 @@ os:
 
 cache: cargo
 
-# Once Travis supports a version of Ubuntu Disco or newer we can apt install QEMU for RISC-V
+# Once Travis supports a version of Ubuntu that inlcudes QEMU 5.1+ we can apt install QEMU for RISC-V
```

Comment:
Nitpick: this line is 101 characters long. We should probably line-wrap it at 80 characters.